sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -75,7 +82,7 @@ object ScalaNativePluginInternal {
     }
   )
 
-  lazy val scalaNativeConfigSettings: Seq[Setting[_]] = Seq(
+  def scalaNativeConfigSettings: Seq[Setting[_]] = Seq(
```

Comment:
One tiny detail I missed earlier, sorry: we can make that again a `lazy val` rather than a `def`, which will speed up build loading.